components/do_verify_email/index.js

```diff
@@ -7,6 +7,16 @@ import {verifyUserEmail} from 'mattermost-redux/actions/users';
 
 import DoVerifyEmail from './do_verify_email.jsx';
 
+function mapStateToProps(state, ownProps) {
+    const config = state.entities.general.config;
+    const siteName = config.SiteName;
+
+    return {
+        ...ownProps,
```

Comment:
this is not needed here, please remove it.

ownProps is being passed regardless of doing it here

Reply:
@jwilander pointed this out in one of the parallel reviews. I'm queueing up all this feedback and will address across all the reviews.